libs/sdk-backend-spi/src/workspace/styling/theme.ts

```diff
@@ -326,82 +778,17 @@ export interface ITheme {
     /**
      * Button customizable UI properties
      */
-    button?: {
```

Comment:
Should it be named rather "IThemeButton" to make it consistent with the previous parts?

Reply:
All are renamed as recommended.